### PR TITLE
Add one example to gdal_utils

### PR DESCRIPTION
**Pull request by agila5**
Following the discussion in #1213, I added one example at gdal_utils help page. I added the \dontrun tag so that it doesn't create a new object everytime it tests the examples but maybe it would be fine.

### CODE REVIEW

**Comment by edzer** [comment]
Thanks; for the same reason I did that too, this afternoon, see here: https://github.com/r-spatial/sf/blob/wkt2/R/gdal_utils.R#L35-L39 (where I was, mistakenly, still working in branch wkt2)

Please let me know how my example can be further improved.

**Comment by agila5** [comment]
I would add both examples since they show two different options of vectortranslate utils (and I can change my example using the same tempfile structure as your example if you want) and I would add an example of another utils, maybe something like:

``` r
geomatrix_path <- system.file("tif/geomatrix.tif", package = "sf")

gdal_utils(
	util = "info",
	source = geomatrix_path, 
	options = c("-json")
)
```
I can create another commit with this example if you agree.

**Comment by edzer** [comment]
Good idea!

**Comment by agila5** [comment]
I've just update the examples following your suggestions.

File: R/gdal_utils.R [hunk 1]
```diff
@@ -33,22 +33,43 @@ resampling_method = function(option = "near") {
 #' @export
 #' @examples
 #'
+#' # info utils can be used to list information about about a raster
+#' # dataset. More info: https://gdal.org/programs/gdalinfo.html
+#' in_file <- system.file("tif/geomatrix.tif", package = "sf")
+#'
 #' \dontrun{
```
Comment (edzer) [inline R/gdal_utils.R]: why do we need a \dontrun{} here?

Comment (agila5) [reply R/gdal_utils.R]: I forgot it from previous example 😅

**Comment by edzer** [comment]
Great, thanks!

**Pull request merged**